commit 9a262d5086bdc5b56f9c91a69e07a2c1d45d83d3 (refs/changes/62/3862/5)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2018-05-02T09:48:14-07:00 (1 year, 5 months ago)
    
    TRITON-335 Better "latest" links for SmartDataCenter and platform
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/tools/mantaput-bits b/tools/mantaput-bits
index 23043c0..7acc80f 100755
--- a/tools/mantaput-bits
+++ b/tools/mantaput-bits
@@ -122,9 +122,13 @@ done
 # upload the md5sums and config.mk
 echo -e $md5sums | mput -v -H 'content-type: text/plain' /${MANTA_USER}${UPLOAD_BASE_DIR}/${upload_subdir}/md5sums.txt
 mput -v -f $BITS_DIR/config.mk /${MANTA_USER}${UPLOAD_BASE_DIR}/${upload_subdir}/config.mk
-# now update the latest link
 
+# now update the branch latest link
 echo "/${MANTA_USER}${UPLOAD_BASE_DIR}/${upload_subdir}" | mput -v -H 'content-type: text/plain' /${MANTA_USER}${UPLOAD_BASE_DIR}/${UPLOAD_BRANCH}-latest
+# If this is a bi-weekly release branch, also update latest-release link
+if [[ $UPLOAD_BRANCH =~ ^release- ]]; then
+    echo "/${MANTA_USER}${UPLOAD_BASE_DIR}/${upload_subdir}" | mput -v -H 'content-type: text/plain' /${MANTA_USER}${UPLOAD_BASE_DIR}/latest-release
+fi
 
 echo "Uploaded to /${MANTA_USER}${UPLOAD_BASE_DIR}/${upload_subdir}"
 
